### PR TITLE
Task/dtserwthre 56 create transfer ppc ui tests

### PR DESCRIPTION
**Pull request by malaw-hw**

### CODE REVIEW

File: UITests/BaseTests/BaseTest.swift [hunk 1]
```diff
@@ -33,8 +33,12 @@ class BaseTests: XCTestCase {
         mockServer = HyperwalletMockWebServer()
         mockServer.setUp()
 
+//        mockServer.setupStub(url: "/rest/v3/users/usr-token/authentication-token",
+//                             filename: "AuthenticationTokenResponse",
+//                             method: HTTPMethod.post)
+
```
Comment (jsaini-hw) [inline UITests/BaseTests/BaseTest.swift:39]: delete commented code

File: UITests/Helper/TransferFunds/TransferFunds.swift [hunk 2]
```diff
@@ -45,7 +45,7 @@ class TransferFunds {
 
     var noteLabel: String
 
-    let title = "Transfer funds"
+    // let title = "Transfer funds"
```
Comment (jsaini-hw) [inline UITests/Helper/TransferFunds/TransferFunds.swift:48]: delete commented code

File: UITests/Responses/TransferFunds/PrepaidCardSecondaryResponse.json [hunk 3]
```diff
@@ -0,0 +1,62 @@
+{
+    "count": 2,
+    "offset": 0,
+    "limit": 10,
+    "data": [
+        {
+            "token": "trm-token",
```
Comment (jsaini-hw) [inline UITests/Responses/TransferFunds/PrepaidCardSecondaryResponse.json:7]: update to `trm-token-001`

File: UITests/Responses/TransferFunds/PrepaidCardSecondaryResponse.json [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+            "primaryCardToken": "trm-0000",
```
Comment (jsaini-hw) [inline UITests/Responses/TransferFunds/PrepaidCardSecondaryResponse.json:8]: update to `trm-token-002`

File: UITests/Responses/TransferFunds/PrepaidCardSecondaryResponse.json [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+                {
+                    "params": {
+                        "rel": "self"
+                    },
+                    "href": "https://localhost.com/rest/v3/users/usr-token/prepaid-cards/trm-token"
+                }
+            ]
+        },
+        {
+            "token": "trm-token",
```
Comment (jsaini-hw) [inline UITests/Responses/TransferFunds/PrepaidCardSecondaryResponse.json:31]: Update to `trm-token-002`

File: UITests/Responses/TransferFunds/PrepaidCardSecondaryResponse.json (hunk 4 above)
Comment (jsaini-hw) [reply UITests/Responses/TransferFunds/PrepaidCardSecondaryResponse.json:8]: primary card token in secondary card response should be a token of primary card

File: UITests/Helper/TransferFunds/TransferFunds.swift [hunk 6]
```diff
@@ -88,6 +131,7 @@ class TransferFunds {
 
     func enterNotes(description: String) {
         notesDescriptionTextField.clearAndEnterText(text: description)
+        //notesDescriptionTextField.typeText(description)
```
Comment (ddhar-hw) [inline UITests/Helper/TransferFunds/TransferFunds.swift:134]: same here ^^

File: UITests/TransferFunds/TransferUserFundsConfirmationTest.swift [hunk 7]
```diff
@@ -270,45 +275,45 @@ class TransferUserFundsConfirmationTest: BaseTests {
...
+     XCTAssertFalse(transferFundsConfirmation.summaryFee.exists)
 
-        let button = transferFundsConfirmation.scheduleTable.buttons["scheduleTransferLabel"]
-        app.scroll(to: button)
-        XCTAssertTrue(button.exists)
-    } */
+     let button = transferFundsConfirmation.scheduleTable.buttons["scheduleTransferLabel"]
+     app.scroll(to: button)
+     XCTAssertTrue(button.exists)
+     } */
```
Comment (ddhar-hw) [inline UITests/TransferFunds/TransferUserFundsConfirmationTest.swift:316]: Can we delete this test if its not being used?

**Review by jsaini-hw — DISMISSED** [review]

**Review by ddhar-hw — APPROVED** [review]

**Review by jsaini-hw — APPROVED** [review]

**Pull request merged**